docs/en/01_configuration.md

```diff
@@ -143,71 +150,72 @@ environment. In this environment, RealMe provide all SSL certificates required t
 
-You should now be able to proceed to testing the standard login form, or [using the RealMe templates](templates.md).
+You should now be able to proceed to testing the standard login form, or [using the RealMe templates](./02_templates.md).
 
-### ITE: Integration Test Environment
+### ITE: integration test environment
 
 - Complete an integration to MTS.
 - You will need a secure certificate which meets the requirements as seen on the [Certificate requirements](https://developers.realme.govt.nz/how-realme-works/certificate-requirements/) page.
-    - If you are using the Common Web Platform, once you have the certificate raise a ticket on the [CWP Service desk](https://www.cwp.govt.nz/service-desk/new-request/) to get it installed.
-    - Otherwise, you can generate one yourself and install it into your test or staging environment.
+  - If you are using the Common Web Platform, once you have the certificate raise a ticket on the [CWP Service desk](https://www.cwp.govt.nz/service-desk/new-request/) to get it installed.
+  - Otherwise, you can generate one yourself and install it into your test or staging environment.
 - Request an account on the [RealMe Developers site](https://developers.realme.govt.nz/), and complete an integration request for ITE.
 - Publish your site to your test or staging environment with a working configuration (`realme.yml` file) for ITE.
 
-### PROD: Production Environment
+### PROD: production environment
 
 - Complete an integration to MTS and ITE.
 - Follow the steps as for the ITE environment above, but creating an integration request for production rather than ITE.
 
 ## More complex environments
+
 If you are working with multiple website environments (e.g. multiple test sites or similar), you will encounter issues
 using the basic configuration system above, because you will want multiple different websites to point to different RealMe
 integrations (e.g. 'test1' website points to MTS, while 'test2' and 'staging' websites both point to ITE). To do this,
 each website installation needs a separate RealMe integration (that is, a different SP Entity ID). Once these are
 configured with RealMe, you can switch them out by modifying configuration. For now (until a solution to #26 is built),
-the best solution is to use `RealMeService::config()->set()` for SS4. **Note:** This can cause a significant performance impact in SilverStripe 4, as by default YML configuration is immutable, and this overrides this which is not ideal. Use this only if it's really necessary, and ensure that the majority of your environments are still configured via YML as normal.
+the best solution is to use `RealMeService::config()->set()` for SS4. **Note:** This can cause a significant performance impact in Silverstripe 4+, as by default YML configuration is immutable, and this overrides this which is not ideal. Use this only if it's really necessary, and ensure that the majority of your environments are still configured via YML as normal.
```

Comment:
Remove stuff that references old versions.

Reply:
Updated